test/functional/mempool_compatibility.py

```diff
@@ -68,8 +68,7 @@ def run_test(self):
         self.log.info("Add unbroadcasted tx to mempool on new node and shutdown")
         unbroadcasted_tx_hash = new_wallet.send_self_transfer(from_node=new_node)['txid']
         assert unbroadcasted_tx_hash in new_node.getrawmempool()
-        mempool = new_node.getrawmempool(True)
-        assert mempool[unbroadcasted_tx_hash]['unbroadcast']
+        assert new_node.getmempoolentry(unbroadcasted_tx_hash)['unbroadcast']
```

Comment:
22707 - partial due to missing changes for `test/functional/mempool_package_limits.py` (which is introduced by bitcoin#21800)

Reply:
adding bitcoin#21800